android.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eu
+#
+# Replace jdk.table xml file
+# For Android Studio to detect the correct JDK path
+# we need to preset the path in the jdk.table.xml
+#
+
+OPTIONS_DIR="$HOME/.AndroidStudio2.2/config/options"
+
+if [[ ! -f ${OPTIONS_DIR}/jdk.table.xml ]]; then
+  mkdir -p ${OPTIONS_DIR}
```

Comment:
Without quoting, this won't work for users with spaces in the path to their home directory:

```sh
mkdir -p "${OPTIONS_DIR}"
```